libbeat/outputs/amqp/batchtracker.go

```diff
@@ -116,4 +135,6 @@ func (b *batchTracker) finalize() {
 
 	b.logger.Debugf("batch completed successfully")
```

Comment:
I think it would be useful to add the values of `counter` and `dropped` to this to help with diagnosing message drops.